libs/auth-api-lib/src/lib/delegations/delegations.service.ts

```diff
@@ -361,7 +366,7 @@ export class DelegationsService {
 
     return uniqBy(
       ([] as DelegationDTO[]).concat(...delegationSets),
-      'fromNationalId',
+      (delegation) => [delegation.domainName, delegation.fromNationalId].join(),
```

Comment:
Let's consider if someone has a natural delegation and a custom delegation from the same national ID.

Before, the natural one would override the custom delegation. With this change it will show up twice in the list with unknown behaviour. We have a task to merge delegations properly, but I would consider this a regression until then.

Can you think of a way to keep the old behaviour for now?

Reply:
Reverted this change, limit the custom delegations to the default domain for now.

We probably need to sort out the delegation merge-ing to do this properly.

Reply:
Our partner organisation is interested to start testing this on dev as soon as next sprint. Happy to merge this to unblock other work, but we need to either filter the delegations based on the client domain or finish the delegation merging before next sprint. 👍